guides/v2.2/javascript-dev-guide/widgets/widget-row-builder.md

```diff
@@ -9,7 +9,7 @@ The RowBuilder [widget](https://glossary.magento.com/widget/) allows you to buil
 
 The RowBuilder widget can be used only on the frontend area.
 
-The RowBuilder widget source is [lib/web/mage/row-builder.js][].
+The RowBuilder widget source is [lib/web/mage/row-builder.js]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Theme/view/frontend/web/js/row-builder.js).
```

Comment:
Revert this change and fix the link in _Link Definitions_ (at the end of file).

Reply:
yep, good point

**fixed**